lib/internal/repl.js

```diff
@@ -55,15 +55,26 @@ function createRepl(env, opts, cb) {
   }
 
   const repl = REPL.start(opts);
-  if (opts.terminal && env.NODE_REPL_HISTORY !== '') {
```

Comment:
That was how I had originally done this, but changed it at @cjihrig's request. I'm fine doing it either though